### DISCUSSION TITLE
SketchFab and EMu. Why can't I use it at the Museum?

### DISCUSSION THREAD

**Discussion by magpiedin** [fieldmuseum/EMu-Documentation#720] [FAQs] [answered]
[This workflow (google doc)](https://docs.google.com/document/d/17f2wdiyn6pIMM8SNnROko6euaUnLFyizJGur0GjJKCA/) provides a way to migrate past/existing content off of SketchFab and to other FMNH-supported modes of presenting 3D media - e.g. [using EMu to document 3D models](https://github.com/fieldmuseum/EMu-Documentation/discussions/335), and publish them to fieldmuseum.org and/or MorphoSource.org

**NOTE:  FMNH will not use SketchFab.com for presenting 3D FMNH media as of March 2024.** 

SketchFab’s [terms of use](https://sketchfab.com/terms) directly conflict with [FMNH Collections Data and Media Norms](https://www.fieldmuseum.org/science/research/use-collections-data-and-images), under which the standard license for FMNH Collections media is “CC BY-NC 4.0” requiring attribution and limiting usage to non-commercial purposes.  For further info, see “Why” in the discussion below, or [in the workflow](https://docs.google.com/document/d/17f2wdiyn6pIMM8SNnROko6euaUnLFyizJGur0GjJKCA/edit#heading=h.j0fajlvf467s).

**Comment by magpiedin** [comment, marked as answer]
**ANSWER:** Use [this workflow to document 3D models in EMu](https://github.com/fieldmuseum/EMu-Documentation/discussions/335), and publish them to [fieldmuseum.org](https://www.fieldmuseum.org) and/or [MorphoSource.org](https://www.MorphoSource.org)

**Comment by magpiedin** [comment]
### Why the restriction on FMNH use of SketchFab?
SketchFab terms of use conflict with FMNH Data Norms, under which the standard license for FMNH Collections media is “CC BY-NC 4.0” requiring attribution and limiting usage to non-commercial purposes. Some of SketchFab's conflicting terms – which would allow for commercial use, without attribution, and with no further coordination with the museum and stakeholders – are highlighted in bold here:

From https://sketchfab.com/terms#user-content:
("Effective February 17, 2024" - [web-archive version here](https://web.archive.org/web/20250802083242/https://sketchfab.com/terms#user-content))


> 4.2.1 Use by us [SketchFab].
> 
> (a) License to Provide and Improve the Services. **By using the Services, you grant a worldwide, non-exclusive, royalty-free, fully paid-up, perpetual, irrevocable, sub-licensable (through multiple tiers) right and license to Sketchfab to use, reproduce, publicly display, adapt, modify, reformat, and distribute User Content as necessary to , improve and promote the Services;** provided that to the extent User Content includes personal data, we shall not use such personal data for any purpose other than providing the Services and as further set forth in our Privacy Policy at https://sketchfab.com/privacy. **You further grant Sketchfab the right to use your name and trademarks, if any, in connection with our use of your publicly shared User Content** for the purposes set forth in this Section 4.2.1.
> 
> (b) License for Safety and Discovery Tools. We use automation to moderate content and improve discovery. This helps us provide a safe and trustworthy service, enforce our rules and make it as easy as possible for you to discover the quality content that you are looking for. Sketchfab and our affiliate, Epic Games, Inc. (“Epic”), use User Content to improve (1) our internal safety tools (“Safety Tools”) to ensure all of our products and services comply with Epic’s Content Guidelines and Epic Community Rules and (2) our search and recommendation tools to improve users’ ability to discover content across all of our products and services ((1) and (2), collectively, “Safety and Discovery Tools”).
> 
> **You additionally grant a royalty-free, perpetual, world-wide, fully sub-licensable (through multiple tiers) license to Epic** limited to using, copying, editing, modifying, inputting, and integrating User Content into and in connection with the development and testing of Epic’s Safety and Discovery Tools.

**Comment by rondlg** [comment]
More reasons why we don't use Sketchfab: https://www.404media.co/email/c98f7ffc-8a7a-4244-99e6-ef97b080402e/?ref=daily-stories-newsletter
